api/v1/models/newsletter.py

```diff
@@ -1,17 +1,28 @@
-from sqlalchemy import Column, String, Text
+from sqlalchemy import Column, String, Text, Integer, DateTime, Boolean, ForeignKey
 from sqlalchemy.orm import relationship
-from api.v1.models.base import user_newsletter_association
 from api.v1.models.base_model import BaseTableModel
+from datetime import datetime, timezone
 
 
 class Newsletter(BaseTableModel):
-    """
-    Newsletter db model
-    """
     __tablename__ = 'newsletters'
 
-    email = Column(String(150), unique=True, nullable=False)
-    title = Column(String, nullable=True)
+    id = Column(Integer, primary_key=True)
+    title = Column(String(100), nullable=False)
+    description = Column(Text, nullable=True)
     content = Column(Text, nullable=True)
+    created_at = Column(DateTime, default=datetime.now(timezone.utc))
+    updated_at = Column(DateTime, default=datetime.now(timezone.utc), onupdate=datetime.now(timezone.utc))
 
-    subscribers = relationship("User", secondary=user_newsletter_association, back_populates="newsletters")
\ No newline at end of file
+    newsletter_subscribers = relationship("NewsletterSubscriber", back_populates="newsletter")
+
+
+class NewsletterSubscriber(BaseTableModel):
+    __tablename__ = 'newsletter_subscribers'
+
+    id = Column(Integer, primary_key=True)
+    email = Column(String(120), nullable=False)
+    newsletter_id = Column(Integer, ForeignKey('newsletters.id'))
+    subscribed_at = Column(DateTime, default=datetime.now(timezone.utc))
```

Comment:
Not really needed as there will be a created_at field normally because of the model inheriting from the BaseTableModel